algorithms.tex

```diff
@@ -1560,10 +1563,10 @@ \section*{25.?? Tag specifiers\hfill[alg.tagspec]}\label{alg.tagspec}
 \pnum
 \requires \tcode{Function} shall meet the requirements of
 \tcode{MoveConstructible}~(\changed{Table~\cxxref{moveconstructible}}{
-\ref{concepts.lib.corelang.moveconstructible}}).
+\ref{concepts.lib.object.moveconstructible}}).
 \enternote \tcode{Function} need not meet the requirements of
 \tcode{CopyConstructible}~(\changed{Table~\cxxref{copyconstructible}}{
-\ref{concepts.lib.corelang.copyconstructible}}).\exitnote
+\ref{concepts.lib.object.copyconstructible}}).\exitnote
```

Comment:
I strongly feel that Destructible through Regular should be in their own subsection due to the refinements - they are all closely related. I have no strong feelings about the title of that subsection. I did change it from "Value-Semantic Object Concepts" to "Object Concepts" because the structibles no longer have a strong value-semantic character, but I've decided that labeling them as "feature concepts" as I did in that earlier comment is not precisely correct. They provide semantic consistency guarantees making them stronger than the feature concepts in the Standard, e.g., CopyConstructible guarantees destruction, &t == addressof(t), allocation.

"Object concepts" is vague. I'm completely open to an alternate name for this family of concepts but I have no better ideas.